crates/ty_server/src/server/api/requests/inlay_hints.rs

```diff
@@ -70,3 +70,10 @@ impl BackgroundDocumentRequestHandler for InlayHintRequestHandler {
 }
 
 impl RetriableRequestHandler for InlayHintRequestHandler {}
+
+fn inlay_hint_type(inlay_hint_content: &InlayHintContent) -> lsp_types::InlayHintKind {
```

Comment:
Nice! Would you mind to also wire it up in wasm and the playground?

https://github.com/astral-sh/ruff/blob/859475f017c3295ba2dbac144dcefdb2a2318250/crates/ty_wasm/src/lib.rs#L980-L987

and here

https://github.com/astral-sh/ruff/blob/859475f017c3295ba2dbac144dcefdb2a2318250/playground/ty/src/Editor/Editor.tsx#L410-L416